config/locales/en.yml

```diff
@@ -30,4 +30,21 @@
 # available at https://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  activerecord:
+    errors:
+      models:
+        user:
+          attributes:
+            password:
+              too_short: 'Password is too short (min is %{count} characters)'
+            password_confirmation:
+              confirmation: "Password not match"
```

Comment:
I think we should use the same type of quotes for all values

Reply:
My mistake. Fixed on 272263f7fb36e0cdf77e88326c4ec84b3f431400